openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -25,6 +25,12 @@ class CourseOverview(django.db.models.Model):
     a course as part of a user dashboard or enrollment API.
     """
 
+    # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
+    VERSION = 1
```

Comment:
Right, but we'd have to manually version. What I want to know is if there's a way of automatically versioning.